src/api/app/controllers/webui/request_controller.rb

```diff
@@ -101,13 +101,25 @@ def show
 
       @diff_limit = params[:full_diff] ? 0 : nil
       @diff_to_superseded_id = params[:diff_to_superseded]
-      @actions = @bs_request.webui_actions(filelimit: @diff_limit, tarlimit: @diff_limit, diff_to_superseded: @diff_to_superseded, diffs: false)
-      @action = @actions.first
+
+      # Handling request actions
+      action = @bs_request.bs_request_actions.first
```

Comment:
:thought_balloon: You can merge those two lines as `action` is not used anywhere else, right?